dts/arm/nordic/nrf5340_cpuappns.dtsi

```diff
@@ -48,6 +48,24 @@
 			 */
 			#include "nrf5340_cpuapp_peripherals.dtsi"
 		};
+
+		/* Cryptocell is a secure-only peripheral, but the availability
+		 * of this peripheral and the cryptocell variant is still usable
+		 * information for the non-secure application which can access
```

Comment:
```suggestion
		 * of this peripheral and the cryptocell variant is still useful
		 * information for the non-secure application as it can access
```

Reply:
"usable information" sounds weird.